content/docs/apps/experimental/restricting-users-to-trusted-ip-ranges.md

```diff
@@ -20,5 +20,60 @@ To fully limit access to your apps and services, you must ensure that all users
 
 ## Restricting access to your own applications
 
-To limit requests to your own applications, you can modify your application logic directly. Alternatively, you can [can follow this example](https://github.com/alphagov/re-paas-ip-safelist-service#quick-demo) within cloud.gov to [create a route service](https://docs.cloudfoundry.org/services/route-services.html#introduction) 
-to block all traffic that does not originate from a trusted IP address range.
+Limiting or restricting access for both ingress and egress traffic from your applications is fairly straightforward through a feature called [Application Security Groups](https://docs.cloudfoundry.org/concepts/asg.html), or ASGs. ASGs support white- and black-listing network traffic from given IPv4 ranges.
+
+### Scoping
+
+Application Security Groups (ASGs) support two types of scoping:
+
+* Platform
+* Space
+
+These scopes allow both operators and developers to have more fine grained access control. Platform-scoped ASGs are applied at the platform level, and would apply to all tenants of cloud.gov. The cloud.gov team does not apply platform-scoped ASGs as they can interrupt tenant operations unless they are needed to meet platform compliance requirements.
+
+Space-scoped ASGs are specific to a given tenant's Space within their allotted organization. This allows for tenants to create Space-specific security groups that apply to all applications within a given space. To apply a Space-scoped ASG to a single application, create a new Space, assign the ASG, and then deploy the application.
+
+### Defaults
+
+cloud.gov is preconfigured with two ASGs: `public_networks` and `dns`. These ASGs are applied by default to all containers in your deployment.
+
+* `public_networks`: This group allows access to public networks, and blocks access to private networks and link-local addresses. Cloud Foundry blocks outgoing traffic to the following IP address ranges by specifically allowing traffic to all other addresses.
+
```

Comment:
Believe it or not, compliance rules forbid us to publish internal network CIDRs in docs even when they're private ranges. 😞 So better to just say that private IP ranges are blocked and let our readers look up what private ranges are, removing the following lines.